book/src/testnet-validator.md

```diff
@@ -160,3 +163,20 @@ If you see any `ERRO` (error) logs, please reach out on
 issue](https://github.com/sigp/lighthouse/issues/new).
 
 Happy staking!
+
+
+## Custom directories
+
+Users can override the default Lighthouse data directories (`~/.lighthouse/{testnet}`) using the `--datadir` flag. The custom data directory mirrors the structure of any testnet specific default directory (e.g. `~/.lighthouse/medalla`).
+
+> Note: Users should specify different custom directories for different testnets. 
```

Comment:
I think it would be good to add this note, or something similar to the help section of the  `--datadir` flag